### PR TITLE
Adding the current and wip validator results to the manifest.json

### PR DESCRIPTION
**Pull request by marcovisserFurore**

### CODE REVIEW

File: validator/manifest.json [hunk 1]
```diff
@@ -2228,168 +2368,182 @@
...
+        "errorCount": 0,
+        "output": [
           "WARNING: Practitioner.meta.profile[0]: StructureDefinition reference \"https://example.de/fhir/StructureDefinition/MyPractitioner\" could not be resolved"
         ]
       }
     },
     {
-      "name" : "observation-triglyceride-good",
-      "file" : "observation-triglyceride-good.xml",
-      "warnings" : [
```
Comment (mmsmits) [inline validator/manifest.json:2252]: Missing element: "warnings"

File: validator/manifest.json [hunk 2]
```diff
@@ -1058,851 +1174,883 @@
...
     },
     {
-      "name" : "ccda-example",
-      "file" : "ccda-example.xml",
-      "use-test" : false,
-      "allowed-extension-domain" : "http://hl7.org/fhir/ccda/StructureDefinition/",
-      "errorCount" : 0,
-      "profile" : {
-        "source" : "ccda-profile.xml",
-        "errorCount" : 0
```
Comment (mmsmits) [inline validator/manifest.json:1432]: missing element: errorCount

File: validator/manifest.json [hunk 3]
```diff
@@ -1058,851 +1174,883 @@
...
+        "output": []
       }
     },
     {
-      "name" : "uk-nhs-pat",
-      "file" : "uk-nhs-pat.xml",
-      "version" : "1.0",
-      "use-test" : false,
-      "allowed-extension-domain" : "http://hl7.org/fhir/ccda/StructureDefinition/",
-      "errorCount" : 0,
```
Comment (mmsmits) [inline validator/manifest.json:1493]: missing element: errorCount

File: validator/manifest.json [hunk 4]
```diff
@@ -1058,851 +1174,883 @@
...
+      "use-test": false,
+      "allowed-extension-domain": "http://hl7.org/fhir/ccda/StructureDefinition/",
+      "profile": {
+        "source": "uk-nhs-sd-pat.xml"
       }
     },
     {
-      "name" : "entry-resource-bad1",
-      "file" : "entry-resource-bad1.xml",
-      "x-errors" : [
```
Comment (mmsmits) [inline validator/manifest.json:1502]: missing element: x-errors

File: validator/manifest.json [hunk 5]
```diff
@@ -1058,851 +1174,883 @@
...
     {
-      "name" : "uk-nhs-pat",
-      "file" : "uk-nhs-pat.xml",
-      "version" : "1.0",
-      "use-test" : false,
-      "allowed-extension-domain" : "http://hl7.org/fhir/ccda/StructureDefinition/",
-      "errorCount" : 0,
-      "profile" : {
-        "source" : "uk-nhs-sd-pat.xml",
-        "errorCount" : 1
```
Comment (mmsmits) [inline validator/manifest.json:1496]: missing element: errorCount

File: validator/manifest.json [hunk 6]
```diff
@@ -2807,1473 +2945,2822 @@
...
+      "version": "4.0",
+      "profile": {
+        "source": "no/PersonWithTwoAddresstypes.StructureDefinition.xml",
+        "supporting": [
           "no/SimpleExtension.StructureDefinition.xml",
           "no/AddressWithExtension.StructureDefinition.xml",
           "no/AddressWithTwoExtensions.StructureDefinition.xml",
           "no/ComplexExtension.StructureDefinition.xml"
         ],
-        "explanatino" : "one about no matcing profiles, then 2 errors for the profiles that con't match",
```
Comment (mmsmits) [inline validator/manifest.json:2950]: explanatino.. -> explanation

File: validator/manifest.json [hunk 7]
```diff
@@ -2807,1473 +2945,2822 @@
...
+        "errorCount": 0,
+        "warningCount": 0,
+        "output": []
       }
     },
     {
-      "name" : "patient-extension-complex-bad1",
-      "file" : "patient-extension-complex-bad1.xml",
-      "version" : "4.0",
-      "erors" : "missing sub-extension",
```
Comment (mmsmits) [inline validator/manifest.json:3051]: erors -> errors 
missing

File: validator/manifest.json [hunk 8]
```diff
@@ -2807,1473 +2945,2822 @@
...
+        "output": [
+          "[ERROR] Instance count for 'Extension.extension' is 0, which is not within the specified cardinality of 1..1 (at Extension.extension)"
+        ]
       }
     },
     {
-      "name" : "patient-extension-complex-bad2",
-      "file" : "patient-extension-complex-bad2.xml",
-      "version" : "4.0",
-      "erors" : "unknown sub-extension",
```
Comment (mmsmits) [inline validator/manifest.json:3064]: erors -> errors
missing

File: validator/manifest.json [hunk 9]
```diff
@@ -2807,1473 +2945,2822 @@
...
+          "[ERROR] Instance count for 'Extension.url' is 0, which is not within the specified cardinality of 1..1 (at Extension.url)"
+        ]
       }
     },
     {
-      "name" : "cda/example",
-      "file" : "cda/example.xml",
-      "version" : "4.0",
-      "explanation" : "can't read a logical model..",
-      "logical" : {
```
Comment (mmsmits) [inline validator/manifest.json:3116]: missing logical and child elements

File: validator/manifest.json [hunk 10]
```diff
@@ -2807,1473 +2945,2822 @@
...
       }
     },
     {
-      "name" : "bundle-validation-location-1.xml",
-      "file" : "bundle-validation-location-1.xml",
-      "version" : "4.0",
-      "explanation" : "This is first of a series of tests that test nevigation and location reporting. This one is straight forward - two independent locations",
-      "java" : {
-        "errorCount" : 2,
-        "error-locations" : [
```
Comment (mmsmits) [inline validator/manifest.json:3558]: missing error-locations

File: validator/manifest.json [hunk 11]
```diff
@@ -2807,1473 +2945,2822 @@
...
-    },
-    {
-      "name" : "type-ref-checked",
-      "file" : "type-ref-checked.xml",
-      "version" : "4.0",
-      "java" : {
-        "errorCount" : 0
       },
-      "profile" : {
-        "assumeValidRestReferences" : true,
```
Comment (mmsmits) [inline validator/manifest.json:3894]: missing assumeValidRestReference

File: validator/manifest.json [hunk 12]
```diff
@@ -2807,1473 +2945,2822 @@
...
     },
     {
-      "name" : "mycommunication.invalid",
-      "file" : "mycommunication.invalid.json",
-      "version" : "4.0",
-      "java" : {
-        "errorCount" : 0
-      },
-      "profile" : {
-        "assumeValidRestReferences" : true,
```
Comment (mmsmits) [inline validator/manifest.json:3909]: missing assumeValidRestReference

File: validator/manifest.json [hunk 13]
```diff
@@ -2807,1473 +2945,2822 @@
...
-    },
-    {
-      "name" : "mycommunication.valid.example",
-      "file" : "mycommunication.valid.example.json",
-      "version" : "4.0",
-      "java" : {
-        "errorCount" : 0
       },
-      "profile" : {
-        "assumeValidRestReferences" : true,
```
Comment (mmsmits) [inline validator/manifest.json:3927]: missing assumeValidRestReference

File: validator/manifest.json [hunk 14]
```diff
@@ -2807,1473 +2945,2822 @@
...
+      "version": "4.0.1",
+      "profile": {
+        "source": "jv-patient-profile-res.xml",
+        "supporting": [
           "jv-patient-profile-dt.xml"
         ],
-        "java" : {
-          "errorCount" : 0,
-          "warningCount" : 1,
-          "debug" : true,
```
Comment (mmsmits) [inline validator/manifest.json:4057]: missing debug element

File: validator/manifest.json [hunk 15]
```diff
@@ -4324,74 +5811,73 @@
...
-      "file" : "mr-single-not-subj-list.xml",
-      "supporting" : [
+      "name": "mr-single-not-subj-list",
+      "file": "mr-single-not-subj-list.xml",
+      "supporting": [
         "mr-m-simple.xml"
       ],
-      "java" : {
-        "errorCount" : 1,
-        "earningCount" : 1,
```
Comment (mmsmits) [inline validator/manifest.json:4360]: missing element earningCount -> warningCount

File: validator/manifest.json [hunk 16]
```diff
@@ -4663,193 +6236,345 @@
...
-          "errorCount" : 1,
-          "output" : [
+      "profile": {
+        "source": "ext-example-nl-profile.json",
+        "java": {
+          "errorCount": 1,
+          "output": [
             "Profile http://fhir.nl/fhir/StructureDefinition/nl-core-address, Element matches more than one slice - LandGBACodelijst, LandISOCodelijst"
-          ],
-          "background" : "see https://chat.fhir.org/#narrow/stream/179166-implementers/topic/why.20does.20this.20.24validate.20fail.3F/near/197350736 - slicing isn't understood by the validator"
```
Comment (mmsmits) [inline validator/manifest.json:4844]: missing element "background"

File: validator/manifest.json [hunk 17]
```diff
@@ -4879,628 +6604,1254 @@
...
+      "firely-sdk-wip": {
+        "errorCount": 0,
+        "warningCount": 0,
+        "output": []
       }
     },
     {
-      "name" : "pat-security-good3",
-      "file" : "pat-security-good3.json",
-      "note" : "Same as good1, but with security checks on",
```
Comment (mmsmits) [inline validator/manifest.json:4927]: missing element "note"

File: validator/manifest.json [hunk 18]
```diff
@@ -4879,628 +6604,1254 @@
...
+        "output": [
+          "[ERROR] Type checking the data: Encountered unknown element 'script' at location 'Patient.name[0].text[0].script[0]' while parsing (at )"
+        ]
       }
     },
     {
-      "name" : "qr-vr",
-      "file" : "qr-vr.json",
-      "version" : "4.0",
-      "description" : "Testing versioned reference to questionnaire",
```
Comment (mmsmits) [inline validator/manifest.json:4992]: missing element: "description"

File: validator/manifest.json [hunk 19]
```diff
@@ -4879,628 +6604,1254 @@
...
+      "supporting": [
         "parameters-profiled-resource-profiles-patient.json",
         "parameters-profiled-resource-profiles-patient2.json"
       ],
-      "profile" : {
-        "source" : "parameters-profiled-resource-profile-params-multiples.json",
-        "java" : {
-          "errorCount" : 1,
-          "warningCount" : 0,
-          "comment" : "This needs to be fixed so that it's not just an error",
```
Comment (mmsmits) [inline validator/manifest.json:5352]: missing element: "comment"

File: validator/manifest.json [hunk 20]
```diff
@@ -5509,440 +7860,892 @@
...
-      "version" : "4.0",
-      "java" : {
-        "errorCount" : 0,
-        "output" : [
-        ]
-      },
-      "profile" : {
-        "source" : "test-input-params.xml",
-        "java" : {
-          "debug" : true,
```
Comment (mmsmits) [inline validator/manifest.json:5681]: missing element: "debug"

File: validator/manifest.json [hunk 21]
```diff
@@ -5509,440 +7860,892 @@
...
+        "output": [
+          "[ERROR] Unable to resolve reference to profile 'http://example.org/fhir/StructureDefinition/modify'. (at Parameters.parameter[0].modifierExtension[0])"
+        ]
       }
     },
     {
-      "name" : "observation-bad-profile",
-      "file" : "observation-bad-profile.xml",
-      "version" : "4.0",
-      "generate-shapshot" : true,
```
Comment (mmsmits) [inline validator/manifest.json:5694]: missing element: "generate-shapshot"

File: validator/manifest.json [hunk 22]
```diff
@@ -5509,440 +7860,892 @@
...
+        "output": [
+          "[WARNING] Instance failed constraint sdf-0 \"Name should be usable as an identifier for the module by machine processing applications such as code generation\" (at StructureDefinition)"
+        ]
       }
     },
     {
-      "name" : "bundle-good-profile",
-      "file" : "bundle-good-profile.xml",
-      "version" : "4.0",
-      "generate-shapshot" : true,
```
Comment (mmsmits) [inline validator/manifest.json:5706]: missing element "generate-shapshot"

File: validator/manifest.json [hunk 23]
```diff
@@ -5509,440 +7860,892 @@
...
+        "output": [
+          "[ERROR] Unable to resolve reference to profile 'http://validitron.unimelb.edu.au/fhir/StructureDefinition/age'. (at Patient.birthDate[0].extension[0])",
+          "[WARNING] Cannot resolve reference Organization/2342 (at Patient.generalPractitioner[0])"
+        ]
       }
     },
     {
-      "name" : "hakan-se",
-      "file" : "hakan-se.json",
-      "fetcher" : "standalone",
```
Comment (mmsmits) [inline validator/manifest.json:5786]: missing element "fetcher"

File: validator/manifest.json [hunk 24]
```diff
@@ -5509,440 +7860,892 @@
...
-        "warningCount" : 1,
-        "infoCount" : 1,
-        "output" : [ "INFORMATION: QuestionnaireResponse.questionnaire: Canonical URL 'http://random.org/fhir/uv/sdc/Questionnaire/54127-6|2015' does not resolve",
-                     "WARNING: QuestionnaireResponse: The questionnaire 'http://random.org/fhir/uv/sdc/Questionnaire/54127-6|2015' could not be resolved, so no validation can be performed against the base questionnaire" ]
-      }
-    },
-    {
-      "name" : "qr-bad-ref2",
-      "file" : "qr-bad-ref2.json",
-      "description": "",
```
Comment (mmsmits) [inline validator/manifest.json:5848]: missing element "description"

File: validator/manifest.json [hunk 25]
```diff
@@ -5509,440 +7860,892 @@
...
-      "fetcher" : "standalone",
```
Comment (mmsmits) [inline validator/manifest.json:5849]: missing element "fetcher"

File: validator/manifest.json [hunk 26]
```diff
@@ -5509,440 +7860,892 @@
...
-        "warningCount" : 1,
-        "infoCount" : 0,
-        "output" : [ "ERROR: QuestionnaireResponse.questionnaire: Canonical URL 'http://hl7.org/fhir/ValueSet/example-extensional' refers to a resource that has the wrong type. Found ValueSet expecting one of [Questionnaire]",
-                     "WARNING: QuestionnaireResponse: The questionnaire 'http://hl7.org/fhir/ValueSet/example-extensional' could not be resolved, so no validation can be performed against the base questionnaire"]
-      }
-    },
-    {
-      "name" : "qr-bad-ref3",
-      "file" : "qr-bad-ref3.json",
-      "description": "",
```
Comment (mmsmits) [inline validator/manifest.json:5862]: see above

File: validator/manifest.json [hunk 27]
```diff
@@ -5509,440 +7860,892 @@
...
-        "warningCount" : 1,
-        "infoCount" : 1,
-        "output" : [ "INFORMATION: QuestionnaireResponse.questionnaire: Canonical URL 'http://hl7.org/fhir/uv/sdc/Questionnaire/54127-6|2015' does not resolve",
-                     "WARNING: QuestionnaireResponse: The questionnaire 'http://hl7.org/fhir/uv/sdc/Questionnaire/54127-6|2015' could not be resolved, so no validation can be performed against the base questionnaire" ]
-      }
-    },
-    {
-      "name" : "supplement-1",
-      "file" : "supplement-obs.json",
-      "description": "Can't use a code defined in a supplement",
```
Comment (mmsmits) [inline validator/manifest.json:5876]: same

File: validator/manifest.json [hunk 28]
```diff
@@ -5509,440 +7860,892 @@
...
-        "errorCount" : 1,
-        "warningCount" : 0,
-        "infoCount" : 0,
-        "output" : [ "Observation.code.coding[0]: CodeSystem http://hl7.org/fhir/test/CodeSystem/supplement-cs is a supplement, so cant be used as a value in Coding.system for 'http://hl7.org/fhir/test/CodeSystem/supplement-cs#test'" ]
-      }
-    },
-    {
-      "name" : "supplement-2",
-      "file" : "supplement-cs.json",
-      "description": "Supplement can't define new codes",
```
Comment (mmsmits) [inline validator/manifest.json:5889]: same

File: validator/manifest.json [hunk 29]
```diff
@@ -5509,440 +7860,892 @@
...
-        "warningCount" : 0,
-        "infoCount" : 0,
-        "output" : [ "ERROR: CodeSystem.concept[0]: The code 'test' is not declared in the base CodeSystem http://loinc.org so is not valid in the supplement" ]
-      }
-    },
-    {
-      "name" : "qr-validation-issue",
-      "file" : "me-qr.json",
-      "supporting" : ["me-vs.json", "me-q.json"],
-      "description": "Reported Issue validating an answer in a questionnaire",
```
Comment (mmsmits) [inline validator/manifest.json:5902]: same

File: validator/manifest.json [hunk 30]
```diff
@@ -5509,440 +7860,892 @@
...
-        "errorCount" : 0,
-        "warningCount" : 0,
-        "infoCount" : 0,
-        "output" : [ ]
-      }
-    },
-    {
-      "name" : "pat-dob-ext",
-      "file" : "pat-dob-ext.json",
-      "description": "Testing primitive extensions",
```
Comment (mmsmits) [inline validator/manifest.json:5914]: same

File: validator/manifest.json [hunk 31]
```diff
@@ -5509,440 +7860,892 @@
...
-        "errorCount" : 1,
-        "warningCount" : 0,
-        "infoCount" : 0,
-        "output" : [ "ERROR: Patient.birthDate.extension[0]: The extension http://validitron.unimelb.edu.au/fhir/StructureDefinition/age is unknown, and not allowed here" ]
-      }
-    },
-    {
-      "name" : "example-organization-search",
-      "file" : "example-organization-search.xml",
-      "description": "ID validation in r2b, and validating unknown code systems against expansions",
```
Comment (mmsmits) [inline validator/manifest.json:5926]: same

File: validator/manifest.json [hunk 32]
```diff
@@ -5509,440 +7860,892 @@
...
-        "output" : [ 
-           "ERROR: Organization.id: id value 'CPR1000991_' is not valid",
-           "ERROR: Organization.address[0].state: The value provided ('Foo') is not in the value set https://fhir.infoway-inforoute.ca/ValueSet/canadianjurisdiction|20170626 (https://fhir.infoway-inforoute.ca/ValueSet/canadianjurisdiction), and a code is required from this value set) (error message = The code provided (http://canadapost.ca/CodeSystem/ProvinceCodes) is not in the value set https://fhir.infoway-inforoute.ca/ValueSet/canadianjurisdiction, and a code is required from this value set. The system Foo is unknown.)"
-        ]
-      }
-    },
-    {
-      "name" : "target-ref-profile-empty",
-      "file" : "target-ref-profile-empty.json",
-      "fetcher" : "standalone",
```
Comment (mmsmits) [inline validator/manifest.json:5940]: missing elelemt "fetcher"

**Review by mmsmits — CHANGES_REQUESTED** [review]

**Review by ewoutkramer — APPROVED** [review]

**Review by mmsmits — APPROVED** [review]

**Pull request merged**